sqlglot/dialects/snowflake.py

```diff
@@ -445,6 +446,11 @@ class Parser(parser.Parser):
             "LOCATION": lambda self: self._parse_location_property(),
         }
 
+        TYPE_CONVERTER = {
+            # https://docs.snowflake.com/en/sql-reference/data-types-numeric#number
+            exp.DataType.Type.DECIMAL: build_default_decimal_type(precision=38, scale=0),
+        }
```

Comment:
Snowflake docs say _`By default, precision is 38 and scale is 0`_, does this hint that these are parameterized by a system knob? 

From a quick look I didn't find such a feature, but if there is this could cause side effects i.e if the user has increased the precision/scale but SQLGLot tranpsiles to `(38, 0)`.

Reply:
I doubt there is a knob for that, it doesn't really make sense. They also don't mention something.